docs/source/reference/envvars.rst

```diff
@@ -137,6 +137,27 @@ Compilation options
    calls the original Python function instead of a compiled version.  This
    can be useful if you want to run the Python debugger over your code.
 
+.. envvar:: NUMBA_CPU_NAME and NUMBA_CPU_FEATURES
+
+    Override CPU and CPU features detection.
+    By setting ``NUMBA_CPU_NAME=generic``, A generic CPU model is picked
+    for the CPU architecture and the features (NUMBA_CPU_FEATURES) is
```

Comment:
backticks around ``NUMBA_CPU_FEATURES``? and s/features/feature list/, s/is defaulted/defaults/